_docs/info/bhyve_kvm_brand.md

```diff
@@ -128,3 +146,43 @@ omnios# socat TCP-LISTEN:5905,reuseaddr,fork UNIX-CONNECT:/data/zone/bhyve/root/
 > It is intended that future zone management tools incorporate this feature
 > in an easy-to-use way.
 
+## Post Install
+
+Once the image is installed and it reboots, you'll get back to the install menu.
+(Note, you'll have to reconnect after VNC disconnects at the reboot to see this).
+To avoid going into another full reinstall, you need to remove the boot cdrom from 
+the zone config.
+
+```terminal
+omnios# zonecfg -z debian
+zonecfg:debian> remote attr name=cdrom
+zonecfg:debian> verify
+zonecfg:debian> commit
+zonecfg:debian> exit
+```
+
+## Debian specific
+
+One final note on Debian. The install image does not setup (as of Debian 10) a proper
+UEFI boot in the right place for Bhyve to find it, so you'll have to boot into the 
+Debian OS following a few steps from the uefi shell, get the OS up, then copy a file
+in place and run update-grub.
+
+# exit from the Uefi Shell
+```terminal
+Shell> exit
+```
+
+# Now select Boot Maintenance Manager from the Bhyve boot screen
```

Comment:
the markdown formatting here is probably not how i was intent. one `#` is header 1, so very, very large.

Reply:
Yes, I meant to do a numbered list but wasn't sure of the markdown. Could use a little bit of a help there.

Reply:
this might help: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists